content/yaml-basic-configuration/environment-variables.md

```diff
@@ -56,6 +56,8 @@ Here is a list of the built-in environment variables with brief explanations:
 | CM_ARTIFACT_LINKS       | Information about generated build artifacts that is available in post-publishing step. Read more about it below.                                                |
 | CM_WORKFLOW_NAME        | Name of the workflow that is currently being built       
 | CM_ENV                  | Environment file that sets/overrides environment variables at build time
+| CM_TRIGGER_SOURCE       | The source that triggered the build. Values are `webhook`, `schedule`, or `api`
```

Comment:
```suggestion
| CM_TRIGGER_SOURCE       | The source that triggered the build. Possible values are `webhook`, `schedule`, or `api`.
```
Wdyt of adding a brief explanation to each trigger source, e.g. `webhook` (initiated by a webhook event / Git webhook), `schedule` (triggered by a build schedule), or `api` (started manually from the UI or via the REST API).

Reply:
I don't know, I thing the names are self-explanatory 🤷🏻‍♂️
maybe yes, I can just explain the `api` value.

Reply:
Ok, I agree about webhook and schedule. :)